src/main/java/at/hannibal2/skyhanni/features/mining/fossilexcavator/ProfitPerExcavation.kt

```diff
@@ -0,0 +1,45 @@
+package at.hannibal2.skyhanni.features.mining.fossilexcavator
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.mining.FossilExcavationEvent
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.CollectionUtils.sortedDesc
+import at.hannibal2.skyhanni.utils.ItemUtils.itemName
+import at.hannibal2.skyhanni.utils.NEUInternalName
+import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.asInternalName
+import at.hannibal2.skyhanni.utils.NEUItems.getPrice
+import at.hannibal2.skyhanni.utils.NumberUtil
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class ProfitPerExcavation {
+    private val config get() = SkyHanniMod.feature.mining.fossilExcavator
+
+    @SubscribeEvent
+    fun onFossilExcavation(event: FossilExcavationEvent) {
+        if (!config.profitPerExcavation) return
+        val loot = event.loot
+
+        val scrapItem = "SUSPICIOUS_SCRAP".asInternalName()
```

Comment:
Probably move this line before the event, in like 17 or smth

Reply:
fixed